Widget Section for Operations (#387)
Laravel-Backpack/docs: 5.x/crud-operation-list-entries.md

}
```

<a name="widget"></a>
## How to add custom sections(aka. Widgets)

> [Widgets](https://backpackforlaravel.com/docs/5.x/base-widgets) (aka cards, aka charts, aka graphs) provide a simple way to insert blade files into admin panel pages. You can use them to insert cards, charts, notices or custom content into pages. You can find available widget [here](https://backpackforlaravel.com/docs/5.x/base-widgets#default-widget-types).

Reviewer: ```suggestion
[Widgets](https://backpackforlaravel.com/docs/{{version}}/base-widgets) (aka cards, aka charts, aka graphs) provide a simple way to insert blade files into admin panel pages. You can use them to insert cards, charts, notices or custom content into pages. You can use the [default widget types](https://backpackforlaravel.com/docs/{{version}}/base-widgets#default-widget-types) or [create your own custom widgets](https://backpackforlaravel.com/docs/{{version}}/base-widgets#creating-a-custom-widget-type).
```

> Backpack template includes two [sections](https://backpackforlaravel.com/docs/5.x/base-widgets#requirements-1) where you can push widgets:
> 
> * before_content
> * after_content

Reviewer: ```suggestion
Backpack's default template includes two [sections](https://backpackforlaravel.com/docs/{{version}}/base-widgets#requirements-1) where you can push widgets:

* `before_content`
* `after_content`
```
